packages/components/src/components/DetailsSearch.vue

```diff
@@ -150,6 +158,29 @@ export default {
         }
       });
 
+      const eventsById = this.appMap.data.events.reduce((map, e) => {
+        map[e.id] = e.callEvent;
+        return map;
+      }, {});
+
+      this.findings.forEach(({ finding, appMapUri: { fragment } }) => {
+        const { traceFilter } = fragment;
+
+        const events = fragment.traceFilter.split(' ').map((idStr) => {
+          const id = Number(idStr.split(':')[1]);
+          return eventsById[id];
+        });
```

Comment:
IMO one of the major drivers for this feature is to replace the use of the trace filter for navigating events within a finding. We should drop this.